app/react/V2/Routes/Settings/Translations/EditTranslations.tsx

```diff
@@ -225,25 +219,25 @@ const EditTranslations = () => {
 
   useEffect(() => {
     switch (true) {
-      case fetcher.formData?.get('intent') === 'form-submit' && Array.isArray(fetcher.data):
+      case fetcher.data instanceof FetchResponseError:
         setNotifications({
-          type: 'success',
-          text: <Translate>Translations saved</Translate>,
+          type: 'error',
+          text: <Translate>An error occurred</Translate>,
+          details: fetcher.data.json?.prettyMessage ? fetcher.data.json?.prettyMessage : undefined,
         });
         break;
 
-      case fetcher.formData?.get('intent') === 'file-upload' && Array.isArray(fetcher.data):
+      case fetcher.formData?.get('intent') === 'form-submit':
         setNotifications({
           type: 'success',
-          text: <Translate>Translations imported.</Translate>,
+          text: <Translate>Translations saved</Translate>,
         });
         break;
 
-      case fetcher.data instanceof FetchResponseError:
+      case fetcher.formData?.get('intent') === 'file-upload':
         setNotifications({
-          type: 'error',
-          text: <Translate>An error occurred</Translate>,
-          details: fetcher.data.json?.prettyMessage ? fetcher.data.json?.prettyMessage : undefined,
```

Comment:
I was thinking that we lost the error handling here, but I checked against production, and it actually doesn't do anything about errors.

Reply:
I don't follow, currently if the backend returns an error it should be notified. Is that not working?

Reply:
There isn't an error notification. It isn't due to this change; it isn't in production either.